### ISSUE TITLE
[FR] тюнинг спеллов

### ISSUE THREAD

**Issue by desudesutalk** [SthephanShinkufag/Dollchan-Extension-Tools#819]
Я так подозреваю что у спеллов типа `samelines`, `samewords`, `longwords` и `whitespace` есть какие-то внутренние константы (типа сколько одинаковых строк это спам).

Частенько бывает что они срабатывают там, где мне бы этого не хотелось. Можно добавить в настройку кнопочку где бы я мог эти константы поднять лично для себя. Я лучше буду изредка видеть как спелл не срабатывает, чем эпизодически пропускать из-за скрытия посты.

**Comment by YellowSM** [comment]
А я после многократных опытов/наблюдений вообще  отказался от `#wipe` - уж слишком много "ложных" срабатываний

**Comment by desudesutalk** [comment]
> уж слишком много "ложных" срабатываний

Да, но иногда он нужен. Просто хочется сделать его менее чувствительным.
